src/ts/Detail/DetailHeader.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+export default function DetailHeader(props) {
+  const { offer } = props;
+  return (
+    <div>
+      <div
+        className="image"
+        style={{ backgroundImage: `url(${offer.images.main})` }}
```

Comment:
it's an object so it's ok to have space between brackets (airbnb style guide: https://github.com/airbnb/javascript/tree/master/react#quotes)